libraries/data/source/mcrl2compilerewriter.in

```diff
@@ -22,19 +22,21 @@
 # executable. All files listed in the output are deleted
 # once the rewriter library is no longer needed.
 
-if [ -z "$CXX" ]; then  # Let user choose via $CXX
-  CXX=`which c++`       # Then test for c++
-  if [ $? -ne 0 ]; then
-    CXX=`which clang++` # Then prefer clang over gcc
-  fi
-  if [ $? -ne 0 ]; then
-    CXX=`which g++`     # Test for gcc
-  fi
-  if [ $? -ne 0 ]; then
-    CXX="@CXX@"         # Last resort: use compiler from build.
+# Test for the compiler from build
+CXX=`which @CXX@`
+if [ $? -ne 0 ]; then
```

Comment:
I think the CXX environment variable should always take preference over the compiler used for the build. Using the compiler from the build should really be a last resort, because this is an absolute path on the developer's machine.

Reply:
I have cleaned up the commits belonging to this pull request and have removed all changes that *should* not be required.